pydatalab/src/pydatalab/bokeh_plots.py

```diff
@@ -380,13 +424,38 @@ def selectable_axes_plot(
         and isinstance(df, list)
         and isinstance(df[0], pd.DataFrame)
     ):
-        save_data = Button(label="Download .csv", button_type="primary", width_policy="min")
-        save_data_callback = CustomJS(
-            args=dict(source=source),
-            code=GENERATE_CSV_CALLBACK,
-        )
-        save_data.js_on_click(save_data_callback)
-        plot_columns = [save_data] + plot_columns
+        is_xrd_data = any(col in df[0].columns for col in ["2θ (°)", "intensity", "twotheta"])
+
+        if is_xrd_data:
+            dropdown_menu = [("Download .csv", "csv"), ("Download .xy", "xy")]
```

Comment:
```suggestion
            dropdown_menu = [("Download .csv", "csv"), ("Download .xy (TOPAS-compatible)", "xy_topas"")]
```

Just marking out that this may not work for other software